arcaflow_plugin_rtla/rtla_plugin.py

```diff
@@ -103,49 +137,52 @@ def run_timerlat(
         stats_per_col = []
         found_all = False
 
-        re_notindex = re.compile(r"^Index")
-        re_notdigit = re.compile(r"^\d")
-        re_notall = re.compile(r"^ALL")
+        re_isunit = re.compile(r"^# Time unit is (.+) .*$")
```

Comment:
In this context, the pattern is probably fine; but for clarification, do you really need to collect "any space terminated string" or are "time units" always alphabetic (e.g., `(\w+)`) as suggested by the sample output?

Reply:
So, I just reread the [`re.match()`](https://docs.python.org/3/library/re.html#re.match) documentation, and it turns out that it (unlike [`re.search()`](https://docs.python.org/3/library/re.html#re.Pattern.search)) constrains the match to the beginning of the string, so you don't need the `^` anchor.  And, the `.*$` anchor doesn't accomplish anything, either.

And, I second Dave's suggestion, which yeilds the following simplification:
```suggestion
        re_isunit = re.compile(r"# Time unit is (\w+)")
```